app/scripts/lib/multichain/address.test.ts

```diff
@@ -0,0 +1,103 @@
+import { stripHexPrefix } from '../../../../shared/modules/hexstring-utils';
+import {
+  isEthAddress,
+  normalizeAddress,
+  normalizeSafeAddress,
+} from './address';
+
+type TestAddress = {
+  address: string;
+  normalizedAddress: string;
+  checksumAddress: string;
+};
+
+const ETH_ADDRESSES = [
+  // Lower-case address
+  {
+    address: '0x6431726eee67570bf6f0cf892ae0a3988f03903f',
+    normalizedAddress: '0x6431726eee67570bf6f0cf892ae0a3988f03903f',
+    checksumAddress: '0x6431726EEE67570BF6f0Cf892aE0a3988F03903F',
+  },
+  // Checksum address
+  {
+    address: '0x6431726EEE67570BF6f0Cf892aE0a3988F03903F',
+    normalizedAddress: '0x6431726eee67570bf6f0cf892ae0a3988f03903f',
+    checksumAddress: '0x6431726EEE67570BF6f0Cf892aE0a3988F03903F',
+  },
+];
+
+const NON_EVM_ADDRESSES = [
+  {
+    address: '0xdeadbeef',
+  },
+  {
+    address: 'bc1ql49ydapnjafl5t2cp9zqpjwe6pdgmxy98859v2',
+  },
+];
+
+function toLowerCaseEthAddress(address: string) {
+  return `0x${stripHexPrefix(address).toLowerCase()}`;
```

Comment:
Why are you removing and then adding the prefix here?
```suggestion
  return address.toLowerCase();
```

Reply:
ah yes, that's a leftover, cause I used to also have a `toUpperCaseEthAddress` too, that was wrongly upper-casing the `0x` 🙈 (and I was using the same pattern for both the lower/upper implementation)

So let me use this your suggestion!